scripts/generate.sh

```diff
@@ -1,28 +1,35 @@
 #!/bin/sh
 
 CONTRACTS=${CONTRACTS:-contracts}
+INFLUXDB_DOCS_URL=https://docs.influxdata.com/influxdb
 
 # generate oss contract
 sed -e '/#REF_COMMON_PATHS/{r ./src/common/_paths.yml' -e 'd}' src/oss.yml |
 sed -e '/#REF_COMMON_PARAMETERS/{r ./src/common/_parameters.yml' -e 'd}' |
 sed -e '/#REF_COMMON_SCHEMAS/{r ./src/common/_schemas.yml' -e 'd}' > src/.oss_gen.yml && \
-swagger-cli bundle src/.oss_gen.yml --outfile ${CONTRACTS}/oss.yml --type yaml && \
+swagger-cli bundle src/.oss_gen.yml --type yaml | \
+(sed -e "s|{{% INFLUXDB_DOCS_URL %}}|${INFLUXDB_DOCS_URL}/v2.0|g" > ${CONTRACTS}/oss.yml) && \
```

Comment:
`swagger-cli` was creating the `$CONTRACTS` directory, so now you'll have to add a `mkdir -p $CONTRACTS` above line 6.

Reply:
did you run `make generate-all` as well, or are you manually editing the files in `contracts`?

Reply:
Yes, always `make generate-all`. I could never tolerate that much manual editing.